sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createSupportingUsers.e2e-spec.ts

```diff
@@ -70,6 +76,82 @@ describe("SupportingUserController(e2e)-createSupportingUsers", () => {
       "supportingUserType",
       SupportingUserType.Parent,
     );
+    // Notification record.
+    const createdNotification = await db.notification.findOne({
+      select: {
+        id: true,
+        user: { id: true },
+        messagePayload: true,
+      },
+      relations: { user: true },
+      where: { user: { id: savedApplication.student.user.id } },
+    });
+    expect(createdNotification.messagePayload).toStrictEqual({
+      template_id: supportingUserInformationNotificationMessage.templateId,
+      email_address: savedApplication.student.user.email,
+      personalisation: {
+        lastName: savedApplication.student.user.lastName,
+        givenNames: savedApplication.student.user.firstName ?? "",
+        supportingUserType: "Parents",
+      },
+    });
+  });
+
+  it("Should create a parent supporting user when requested.", async () => {
+    // Arrange
+    const savedApplication = await saveFakeApplication(db.dataSource);
+    const fakeCreateSupportingUsersPayload =
+      createFakeCreateSupportingUsersPayload(savedApplication.id, [
+        SupportingUserType.Parent,
+      ]);
+
+    // Act
+    const result = await supportingUserController.createSupportingUsers(
+      createFakeWorkerJob<
+        CreateSupportingUsersJobInDTO,
+        ICustomHeaders,
+        CreateSupportingUsersJobOutDTO
+      >(fakeCreateSupportingUsersPayload),
+    );
+
+    // Asserts
+    expect(result).toHaveProperty(
+      FAKE_WORKER_JOB_RESULT_PROPERTY,
+      MockedZeebeJobResult.Complete,
+    );
+
+    const updatedApplication = await db.application.findOne({
+      select: { supportingUsers: true },
+      relations: { supportingUsers: true },
+      where: {
+        id: savedApplication.id,
+      },
+    });
+    expect(updatedApplication.supportingUsers).toHaveLength(1);
+    const [supportingUser] = updatedApplication.supportingUsers;
+    expect(supportingUser).toHaveProperty(
+      "supportingUserType",
+      SupportingUserType.Parent,
+    );
+    // Notification record.
+    const createdNotification = await db.notification.findOne({
+      select: {
+        id: true,
+        user: { id: true },
+        messagePayload: true,
+      },
+      relations: { user: true },
+      where: { user: { id: savedApplication.student.user.id } },
+    });
```

Comment:
As this code is repeating, it can be moved to common method in the test file scope.